tests/stub/sessionparameters.py

```diff
@@ -38,7 +38,7 @@
 !: AUTO GOODBYE
 !: AUTO RESET
 
-C: RUN "RETURN 1 as n" {} {"bookmarks": ["b1", "b2"]}
+C: RUN "RETURN 1 as n" {} {"bookmarks{}": ["b1", "b2"]}
```

Comment:
Is this change intended ?

Reply:
Yes. It allows the bookmarks to be sent in arbitrary order.